### PR TITLE
Add isterm flag to AST info

### PR DESCRIPTION
**Pull request by Cisphyx**

### CODE REVIEW

File: synapse/tests/files/stormcov/lookup.storm [hunk 1]
```diff
@@ -3,3 +3,4 @@ function foo() {
     stop
 }
 yield $foo()
+yield $foo()
```
Comment (invisig0th) [inline synapse/tests/files/stormcov/lookup.storm:6]: What's this dood about?

File: synapse/utils/stormcov/plugin.py [hunk 2]
```diff
@@ -112,6 +112,9 @@ def dynamic_source_filename(self, filename, frame, force=False):
         nodeid = id(node)
 
         info = self.node_map.get(nodeid, s_common.novalu)
+        if info is None:
```
Comment (invisig0th) [inline synapse/utils/stormcov/plugin.py:115]: Should this check be fore novalu or am i missing something?

**Review by invisig0th — CHANGES_REQUESTED** [review]

File: synapse/utils/stormcov/plugin.py (hunk 2 above)
Comment (Cisphyx) [reply synapse/utils/stormcov/plugin.py:115]: this checks for novalu next, but if it is None we can skip it

File: synapse/tests/files/stormcov/lookup.storm (hunk 1 above)
Comment (Cisphyx) [reply synapse/tests/files/stormcov/lookup.storm:6]: this is coverage for the `if info is None:` check that was added

Comment (Cisphyx) [reply synapse/tests/files/stormcov/lookup.storm:6]: it needed to call it a second time to hit that check

**Review by invisig0th — APPROVED** [review]

**Pull request merged**